python/ray/tests/test_node_manager.py

```diff
@@ -185,6 +185,54 @@ def f():
     )
 
 
+@pytest.mark.parametrize(
+    "call_ray_start",
+    ["ray start --head --ray-client-server-port=25555"],
+    indirect=True,
+)
+def test_exiting_driver_clears_infeasible(call_ray_start):
```

Comment:
Could you add some comment below describing what's the purpose of this test? Just feel not easy to understand it.